prisma/seed.ts

```diff
@@ -25,20 +32,55 @@ main()
 function getUsers() {
   return [
     {
+      id: '82bf2c53-a9eb-4029-8d6d-17f514e8be7b',
       email: 'me@zainf.dev',
       name: 'Zain',
       phoneNumber: '+6512345678',
       role: ROLES.ADMIN,
     },
     {
+      id: '76ca6998-da6a-4523-bb71-397f7472cd49',
       email: 'vika@rbagi.id',
       name: 'Vika',
       role: ROLES.AUTHOR,
     },
     {
+      id: 'ace81e7d-371a-42f4-99d5-2477237cc23d',
       email: 'pk@zainf.dev',
       name: 'Pejuang Kode',
       role: ROLES.MEMBER,
     },
   ]
 }
+
+function getCourse() {
+  return [
+    {
+      authorId: '76ca6998-da6a-4523-bb71-397f7472cd49',
```

Comment:
Please use `user.id`s generated by Prisma instead.
You can refer to [this code](https://github.com/zainfathoni/kentcdodds.com/blob/3cf8319609aad459b4e8e00352d0931a53104e0b/prisma/seed.ts#L82) as an example.
```suggestion
```